pkg/npm/npm_test.go

```diff
@@ -295,7 +302,7 @@ func TestNpm(t *testing.T) {
 			Utils:   &utils,
 			Options: options,
 		}
-		err := exec.RunScriptsInAllPackages([]string{"foo"}, nil, nil, true)
+		err := exec.RunScriptsInAllPackages([]string{"foo"}, nil, nil, true, nil)
```

Comment:
I would prefer if RunScriptsInAllPackages() would take a single struct as parameter. Then you can add optional fields to the struct as needed, without having to adopt callers.

Reply:
I think we had that discussion before and I agree with you, but based on Daniel's feedback I changed it last time when I refactored the npm pkg. But yeah this is the second time we are adding a parameter to `RunScriptsInAllPackages()` and we would have saved time using the approach you suggested. But I am in favor of leaving this one as it is now, since it is out of scope of this PR to change it again.

Reply:
OK